docs/src/api/make_pages.py

```diff
@@ -0,0 +1,43 @@
+"""Generate the api pages and navigation.
+NOTE: Works best when following the Google style guide
+https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html
+https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings
+"""
+
+import mkdocs_gen_files
+from pathlib import Path
+import os
+import subprocess
+
+package = os.getenv("PACKAGE")
+
+element = package.split("_", 1)[1]
+# if not Path(f"workflow_{element}").is_dir():
+#     try:
+#         subprocess.run(
+#             f"git clone https://github.com/tdincer/workflow-{element.replace('_','-')}.git /main/delete".split(
+#                 " "
+#             ),
+#             check=True,
+#             timeout=5,
+#         )
+#         os.system(f"mv /main/delete/workflow_{element} /main/")
+#         os.system("rm -fR /main/delete")
+#     except subprocess.CalledProcessError:
+#         pass  # no repo found
```

Comment:
Should we remove?

Reply:
Sure, I'll remove. This isn't the latest method of fetching the workflow, which I've since migrated to the docker compose.